Todo.WillowBear/Todo.WillowBear/Data/TodoItemConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Todo.WillowBear.Models;
+
+namespace Todo.WillowBear.Data
```

Comment:
Use new namespaces

Reply:
Could you clarify this one for me please? Should it be in its own namespacE?